flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -304,4 +368,60 @@ public String getDescription() {
 			return "TestOptimizerPlan <input-file-path> <output-file-path>";
 		}
 	}
+
+	public static final class TestExecuteTwice {
+
+		public static void main(String args[]) throws Exception {
```

Comment:
Yes. But marking the class `private` won't interfere with that. I've already pushed a fix and it does pass.